client/src/components/Dashboard.js

```diff
@@ -1,40 +1,69 @@
 import React from 'react';
+import { Button, Icon } from 'semantic-ui-react'
+import DashboardMenu from './DashboardMenu.js'
 
 class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            user: {
+                name: 'Gabriel'
+            }
+        };
+    }
 
-    handleSubmit = (event) => {
+    handleClick = (event, { name }) => {
         event.preventDefault();
-
-        sessionStorage.removeItem("user")
-        window.location.href = '/'
+        window.location.href = `/${name}`;
     }
 
     render() {
+        // COMMENTED OUT FOR DEBUGGING PURPOSES, UNCOMMENT OUT WHEN READY FOR DEPLOYMENT
         // Logic to check if user is already logged in
-        let user = {};
-        let loginStatus = false;
-
-        if (JSON.parse(sessionStorage.getItem("user") !== null)) {
-            user = JSON.parse(sessionStorage.getItem("user"));
-            loginStatus = user.user.isLoggedIn;
-        }
-
-        if (loginStatus) {
-            return (
-                <div className="container">
-                    <div className="ui middle aligned center aligned grid">
-                        <div className="column" style={{ maxWidth: '450px' }}>
-                            <h1>Welcome, {user.user.name}!</h1>
-                            <div className="ui fluid large teal submit button" onClick={this.handleSubmit}>Logout</div>
+        // let user = {};
+        // let loginStatus = false;
+
+        // if (JSON.parse(sessionStorage.getItem("user") !== null)) {
+        //     user = JSON.parse(sessionStorage.getItem("user"));
+        //     loginStatus = user.user.isLoggedIn;
+        // }
+
+        // if (loginStatus) {
+        return (
+            <div className="container">
+                <DashboardMenu page={'dashboard'}></DashboardMenu>
```

Comment:
Simply the string "dashboard"?

Reply:
Err I'm not sure about that, already have a Dashboard component. The DashboardMenu component refers to the top part of the page for all pages it is present in.

Will leave it for now until I think of how to resolve appropriate component naming conventions.

Reply:
As in.... 

```suggestion
                <DashboardMenu page='dashboard'></DashboardMenu>
```

is there a need for the `{}`?